elements/game/renderer.js

```diff
@@ -1,6 +1,6 @@
 const _ = require('lodash')
 const assets = require('../../lib/utils/assets')
-const game = require('../../lib/utils/game')
+const game = require('../../lib/game/index')
```

Comment:
oversight of me, webstorm refactored it like this. Is fixed in my next pull request